### ISSUE TITLE
Please provide detailed compilation instructions 

### ISSUE THREAD

**Issue by JRSV** [essej/aoo#7]
hello,

Deken packages seem to be compiled with a version of GlibC  newer than the one I have in my UbuntuStudio 22.04 (kernel 5.04) so I need to compile the object myself, unfortunately I do not understand how to build with pd_libbuilder. It is installed in my system, but I do not know what to do.

Do I call pd-libbuilder from terminal in the directory of aoo? the make instructions provided produce. I am adding to the make instruction provided the PDLIBDIR=/usr/share/pd-lib-builder

*** No rule to make target 'install'. Stop.

I would appreciate some help with this.

thanks

**Comment by Spacechild1** [comment]
1. that's the wrong repo
2. the Deken packages have been fixed in the meantime

**Comment by essej** [comment]
Yes, this repo is an old fork used ONLY for building the current version of SonoBus. For use of the library, please go to the upstream repository at: https://git.iem.at/aoo/aoo
